d3a_interface/read_user_profile.py

```diff
@@ -388,6 +388,9 @@ def read_profile_without_config(input_profile: Dict, slot_length_mins=15) -> Dic
 
 
 def read_and_convert_identity_profile_to_float(profile):
-    parsed_profile = ast.literal_eval(str(profile))
+    try:
+        parsed_profile = ast.literal_eval(profile)
+    except ValueError:
+        parsed_profile = profile
```

Comment:
Hmm, I don't understand. What is the issue here? Chris is passing a filename string right?

Reply:
According to the ticket this is the problem, and I think that changing the interface library is the wrong way to go. I think that if the infinite_bus.py used the `read_arbitrary_profile` method, then the input would have been detected that it is a filename and not a dict profile.